### DISCUSSION TITLE
Image URL exception

### DISCUSSION THREAD

**Discussion by bybzc** [langgenius/dify#5516] [General]
I deployed a service in the local LAN and used a public router as a port to forward connections to servers within the LAN. After configuring NEXT_PUBLIC_API_PREFIX and NEXT_PUBLIC_PUBLIC_API_PREFIX, I successfully accessed the corresponding IP and entered the system. However, strangely, the URL for image access is still 127.0.0.1. How should I configure it?
![image](https://github.com/langgenius/dify/assets/67107239/e126c053-bcf0-4cb0-b3b1-3c252508c1b4)

**Comment by santime27** [comment]
When you configure your machine or server to deploy Dify on a local LAN, you must first set a static IP address on the machine or server where you want to deploy Dify. You can configure this setting on your router. If you don't do this, the local IP might change when you power off the machine or server. This setting is not required for Dify to work, but it's a best practice.

On the other hand, when you deploy Dify and access it from the same machine, it's normal for the IP to be 127.0.0.1. However, if you want to access Dify from another host located on your LAN, you must use the IP address of the machine or server where Dify was deployed.

I have deployed Dify on my local LAN, so I can help you if you encounter any problems.
